core/src/assets/customResources/customResources2Schema.json

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
Is this needed? This looks like my old file that can be deleted

Reply:
indeed